docs/dns/rrsets.rst

```diff
@@ -542,11 +548,21 @@ Special care needs to be taken with some types of records, as explained below.
 Restricted Types
 ````````````````
 
-``ALIAS``, ``DNAME``
-    These record types are used very rarely in the wild.  Due to conflicts with
-    the security guarantees we would like to give, these record types are
-    disabled in our API.  If you attempt to create such RRsets, you will receive
-    a ``400 Bad Request`` response.
+``ALIAS``
+    Due to conflicts with the security guarantees we would like to give, this
+    record type is disabled in our API.  If you attempt to create such RRsets,
+    you will receive a ``400 Bad Request`` response.
+
+    If you need redirect functionality at the zone apex, consider using the
+    ``HTTPS`` record type which serves exactly this purpose.  (Note that this
+    record type is not yet supported in all browsers.)
```

Comment:
you may want to put a date here

Reply:
I couldn't find anything reliable

Reply:
I meant: as of 06/2021, this is not yet supported in all browsers

Reply:
done